### ISSUE TITLE
Entered ip in AVS app but not seeing login button

### ISSUE THREAD

**Issue by roycerivard13** [alexa/alexa-avs-sample-app#421]
Hi all,

I added my phone's IP into SSL.cnf, and installed the app on my phone (Moto G XT1033 - falcon, running CyanogenMod 14.1 nightly). I typed my computer's "192" IP (192.168.x.x) followed by what I'm assuming is the port, into the Device Address field on the app (so in total it was https://192.168.x.xx:8443 where x.xx is the end of the laptop's IPv4 address - the laptop is running Lubuntu and is the host of the Java applet that I had to "mvn exec:exec" in the terminal). I don't get a gold button. I'm on the same network. I've tried everything. I'm not using the localhost IP - I'm using the one that is broadcast (I tried the one Google tells me when I search "whats my ip", then I tried the IP address in Connection Info and the Broadcast Address in Connection Info as well). Nothing seems to work. Any help is appreciated!

Thanks in advance.

**Comment by max-erler** [comment]
What is the behavior of the app when you enter the IP address? It doesn't show the Login with Amazon button, but does something else happen?

**Comment by roycerivard13** [comment]
No, nothing else happens.                                                                                                                                                                         Sent via the BlackBerry Hub for Android                                                                                                                                                                            From: notifications@github.comSent: December 22, 2016 5:45 PMTo: alexa-avs-sample-app@noreply.github.comReply-to: reply@reply.github.comCc: royriv913@gmail.com; author@noreply.github.comSubject: Re: [alexa/alexa-avs-sample-app] Entered ip in AVS app but not seeing login button (#421)  What is the behavior of the app when you enter the IP address? It doesn't show the Login with Amazon button, but does something else happen?

**Comment by bonhomie** [comment]
I'm getting the same issue @roycerivard13, tried it on both iOS and Android and the connection seems to be failing.

**Comment by keleric** [comment]
On Android, can you make sure that the ca.crt file in src/res/raw is copied from the javaclient certs folder? The two files need to be the same.
